Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.8.10
+RUN apt-get update -y
```

Comment:
Suggest combining the apt invocations to the same RUN with && to reduce the number of layers and clean up with `rm -rf /var/lib/apt/lists/*`

Reply:
Agree; going to do this.